CONTRIBUTING.md

```diff
@@ -30,6 +30,7 @@ The following items will be discussed in this document :
 
 * [Contributing Documentation Changes](#Contributing+Documentation+Changes)
```

Comment:
I don't think the in-doc HREF's work at all. We typically lower case the characters and replace white spaces with dashes, e.g.  `#contributing-...`